lib/config/core.js

```diff
@@ -76,6 +76,7 @@ function load () {
   cb = once(function (er, conf) {
     if (!er)
       exports.loaded = conf
+      loading = false
```

Comment:
Hmm... need a new PR, because this one has already been merged into `npm@2.7.5`.